src/PostgREST/ApiRequest.hs

```diff
@@ -138,8 +140,10 @@ userApiRequest schema req reqBody =
   lookupHeader    = flip lookup hdrs
   hasPrefer val   = any (\(h,v) -> h == "Prefer" && v == val) hdrs
   singular        = hasPrefer "plurality=singular"
-
-
+  representation
+    | hasPrefer "return=representation" = Full
+    | hasPrefer "return=none" = None
```

Comment:
[RFC7240](https://tools.ietf.org/html/rfc7240#page-9) suggests the string "minimal" rather than "none."